src/analytics/src/main/scala/osmesa/analytics/oneoffs/StreamingAOIMonitor.scala

```diff
@@ -174,35 +171,54 @@ object StreamingAOIMonitor
 
             // 2. EXTRACT SALIENT INFO FROM DIFFS
             //    Prepare a dataset of summaries, one for each notification to send.
-            val messageInfo: Dataset[NotificationSummary] = {
+            val changeSummary = {
               val stats = diffs.withDelta
                 .withColumn("measurements", DefaultMeasurements)
                 .withColumn("counts", DefaultCounts)
```

Comment:
Any thoughts about either using this or killing it?

Reply:
Ah true. We're not using this at the moment. I suppose we just delete along with the commented out code and re-add later if that's what ends up being more useful.